### PR TITLE
task_51993_init_project

### PR DESCRIPTION
**Pull request by VuTan115**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/51993)

## WHAT (optional)
- Init Project
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/51993)

### CODE REVIEW

**Comment by VuTan115** [comment]
ready

**Comment by VuTan115** [comment]
ready

**Comment by manhnt-0960** [comment]
ready

**Comment by manhnt-0960** [comment]
ready

**Comment by manhnt-0960** [comment]
commented

**Comment by manhnt-0960** [comment]
ready

File: .sun-ci.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        environment:
+          MYSQL_DATABASE: db_test
+          MYSQL_USER: user_test
+          MYSQL_PASSWORD: password_test
+          MYSQL_ROOT_PASSWORD: password_test
+    before_script:
+      - bundle _2.1.4_ install --path vendor/bundle # _2.1.4_ if ruby 2.7.1 or 3.0.2
+    script:
+      - RAILS_ENV=test bundle exec rails db:drop db:create db:migrate
+      - bundle _2.1.4_ exec rspec # _2.1.4_ if ruby 2.7.1 or 3.0.2
```
Comment (manhnt-0960) [inline .sun-ci.yml]: tạm thời comment những dòng này lại để pass CI

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HotelManagement</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
```
Comment (manhnt-0960) [inline app/views/layouts/application.html.erb]: "

**Comment by VuTan115** [comment]
ready

**Pull request merged**